README.md

```diff
@@ -13,66 +13,113 @@ Assumptions:
 
-| ENV VAR        | VALUE                            |
-|----------------|----------------------------------|
-| USASPENDING_API| https://api.usaspending.gov/api/ |
-| MAPBOX_TOKEN   | ''                               |
-| GA_TRACKING_ID | ''                               |
+| ENV VAR           | DEFAULT VALUE (if not set)       | PURPOSE 
+|-------------------|----------------------------------|----------------------------------------------------|
+| `ENV`             | dev                              | Determine bundling optimizations and feature flags | 
+| `USASPENDING_API` | https://api.usaspending.gov/api/ | API URL to get backend data                        |
+| `MAPBOX_TOKEN`    | ''                               | Product key for use of MapBox features             |
+| `GA_TRACKING_ID`  | ''                               | Google Analytics key for tracking usage in prod    |
 
-### Scripts
+These values can be set in one of 3 methods:
+
+1. In a git-ignored `.env` file, if using `docker` or `docker-compose` to execute scripts 
 
-| Script     | Output                                              |
-|------------|-----------------------------------------------------|
-| `npm start`  | dev server with hot reloading                       |
-| `npm prod`   | generates static assets w/ production optimization  |
-| `npm dev`    | generates static assets w/ development optimization |
-| `npm lint`   | executes linter                                     |
-| `npm test`   | executes unit tests                                 |
-| `npm sass`   | dev server w/ scss source maps                      |
-| `npm travis` | executes travis validation script                   |
-
-### Deployment Build Step with DockerFile
-When deploying we use the dockerfile for our build step. We can simulate that process as follows:
-
-```
-# from usaspending-website root, build the container image
-docker build -t usaspending-website .
-# generate static artifacts in ./public
-docker run -i --rm=true -v $(pwd)/public:/node-workspace/public usaspending-website /bin/sh -c 'npm run dev'
-# mount the static artifacts to an nginx image and run
-docker run -p 8020:80 -v $(pwd)/public:/usr/share/nginx/html:ro nginx
-```
-
-The app should now be running at `http://localhost:8020`.
+    ```bash
+    # .env file
+    USAPENDING_API=http://localhost:8000/api
+    # ... next var
+    ```
+    - Confirm Docker Compose picks up vars by running: `docker-compose config` to see the values applied
+    - Also can be applied with `docker run --env-file .env ...`
+    - _NOTE: Default env var values are set in the `Dockerfile` and will be used in `docker run` if none are provided_
+2. Using `export` to set them in the current executing shell environment.
+
+    ```bash
+    $ export USAPENDING_API=http://localhost:8000/api
+    $ npm dev
```

Comment:
If we're just talking about setting ENV variables, which it appears we are, we should remove the `npm dev` statements from these code snippets.

Reply:
This implies that once the env var is set, the npm script run will _look_ for it. 
I could change to 
    ```bash
    $ export USAPENDING_API=http://localhost:8000/api
    $ # run npm script here that uses this env var
    ```
    
Or just get rid of it. Consider junior developers (on or off this project) trying to get this open source project to run.